src/Sentry.AspNetCore/SentryTracingMiddleware.cs

```diff
@@ -146,9 +152,20 @@ public async Task InvokeAsync(HttpContext context)
                         transaction.Name = transactionName;
                     }
 
-                    transaction.Finish(
-                        SpanStatusConverter.FromHttpStatusCode(context.Response.StatusCode)
-                    );
+                    var status = SpanStatusConverter.FromHttpStatusCode(context.Response.StatusCode);
+                    if (exception == null)
```

Comment:
```suggestion
                    if (exception is null)
```

I thought we switched to this style now?

Reply:
yeah I missed it